src/Xamarin.Android.NamingCustomAttributes/Android.App/ServiceAttribute.cs

```diff
@@ -36,6 +37,8 @@ public ServiceAttribute ()
 
 	public string? Label { get; set; }
 
+	string IJniNameProviderAttribute.Name => Name ?? "";
```

Comment:
For this one, I don't think we can change `IJniNameProviderAttribute.Name` to be a `string?`. I was seeing an API break for that. Seems simpler to just define the interface and check for null.

Reply:
For maintainability could this change go in a partial class ? That way we would not need to modify these files directly when they are regenerated.